Lib/collections/__init__.py

```diff
@@ -203,6 +203,28 @@ def move_to_end(self, key, last=True):
             first.prev = soft_link
             root.next = link
 
+    def sort(self, /, **kwargs):
+        if len(self) > 2:
+            return
```

Comment:
Is this sort method sorts empty or 1-element OrderedDict?

Reply:
I don't know.
Your code is doing `if len(self) > 2: return`.
So following code will be executed only when `len(self)` is 0, 1, or 2.

Reply:
Oh, sure, sorry.